cmd/nginx/flags.go

```diff
@@ -142,7 +142,7 @@ extension for this to succeed.`)
 			`Customized address to set as the load-balancer status of Ingress objects this controller satisfies.
 Requires the update-status parameter.`)
 
-		dynamicCertificatesEnabled = flags.Bool("enable-dynamic-certificates", true,
+		enableDynamicCertificates = flags.Bool("enable-dynamic-certificates", true,
 			`Dynamically update SSL certificates instead of reloading NGINX.
 Feature backed by OpenResty Lua libraries. Requires that OCSP stapling is not enabled`)
```

Comment:
can you edit and delete `Requires that OCSP stapling is not enabled`? it does not make sense anymore

Reply:
done